### PR TITLE
Metadata updates

### PR DESCRIPTION
**Pull request by rasienko**
@arist0tl3 can you take a look at these changes? It's mostly metadata and copy edits, but I did add some info to the video VCC reference and address some user feedback on the custom VCC blueprint. Let me know if you notice anything that's inaccurate. Thanks!

### CODE REVIEW

File: src/asciidoc/docs/customizations/video.adoc [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
 :page-slug: video
+:page-description: Standard VCC for uploading video files with automatic transcoding and formatting.
 
-* {blank}
+* Users can
 //tag::description[]
-Uploaded video is automatically transcoded to be appropriate for playback across devices.
+upload video files.
 //end::description[]
+* Uploaded videos are automatically compressed, transcoded to .mp4 format, and cropped to fit 16:9 portrait for optimal playback across devices.
```
Comment (anderjason) [inline src/asciidoc/docs/customizations/video.adoc:9]: I'm not sure where this is defined exactly, but the platform seems to crop videos differently depending on the original aspect ratio. The resulting video is either 16:9 (landscape) or 9:16 (portrait)

File: src/asciidoc/docs/customizations/video.adoc [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 * Pose detection data can be generated by passing an optional `typeOption`.
 When specified, pose data is available after upload by appending `.poses` to the returned URL.
 For example, `\https://objects.koji-cdn.com/project-id/my-video.mp4.poses`.
```
Comment (anderjason) [inline src/asciidoc/docs/customizations/video.adoc:12]: I'm curious about the format of the poses file, but I don't think it's essential to have that info now. Personally, I haven't used this feature before.

**Review by anderjason — APPROVED** [review]
Looks good to me. I added a few minor comments in the video section.

File: src/asciidoc/docs/customizations/video.adoc (hunk 2 above)
Comment (rasienko) [reply src/asciidoc/docs/customizations/video.adoc:13]: Thanks! Added some info to clarify this point!

**Pull request merged**